packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -360,14 +360,10 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
       }
     }
 
-    const attachmentDownloadResults: AttachmentDownloadResult[] = [];
-    for (const id in options.attachmentUrls) {
-      const response = await fetchWithAuthentication(options.attachmentUrls[id], accessToken);
-      const blob = await response.blob();
-      attachmentDownloadResults.push({ attachmentId: id, blobUrl: URL.createObjectURL(blob) });
-    }
+    const response = await fetchWithAuthentication(options.attachmentUrl, accessToken);
```

Comment:
What happens in a failure case?

Reply:
An error is thrown and can be caught. We have written a few unit tests for those cases, ie missing token credential, error in response etc.

Reply:
And for the case of the download queue, the queue fails?